scripts/lint/lint-markdown.js

```diff
@@ -288,6 +288,20 @@ const opts = {
         MD034: false,
         // Allow bold/italicized paragraphs
         MD036: false,
+        // Images should have alternate text (alt text)
+        MD045: false,
+        // Emphasis style
+        MD049: false,
+        // Strong style
+        MD050: false,
+        // Link fragments should be valid
+        MD051: false,
+        // Link and image reference definitions should be needed
+        MD053: false,
+        // Table pipe style
+        MD055: false,
+        // Table column count
+        MD056: false,
```

Comment:
These are all lints we systemically ignore. We can enable them, but each lint comes with dozens to hundreds of errors and I don't think we need to tackle that in this PR.